src/app/_components/DistrictSearch/actions.js

```diff
@@ -0,0 +1,13 @@
+'use server';
+
+import { prisma } from '@/lib/db';
+
+// export const getDistrictsList = async () => {
+//   const districtsList = await prisma.district.findMany({
+//     orderBy: { name: 'desc' },
+//   });
+//   const defaultOption = { id: 'all-districts', name: 'Усі' };
+//   return [defaultOption, ...districtsList];
+// };
+
+export const getDistrictsList = async () => await prisma.district.findMany();
```

Comment:
🟠 now that looks even weirder, having whole separate file for just replacing 'prisma' call to 'getDistrictList' call 🤔

Reply:
And by the way it doesn't make sense to put it into `async`-function and `await` results before returning:
- `async` function returns promise
- so in your case you just wrap promise into promise

You'd get same result like this:
```javascript
export const getDistrictsList = () => prisma.district.findMany();
```

Reply:
> 🟠 now that looks even weirder, having whole separate file for just replacing 'prisma' call to 'getDistrictList' call 🤔

It's not about just replacing 'prisma' call. This file has 'use server' directive at the top. The idea is to execute this call on the server.

Reply:
Apparently, based on the approach used on the recent dev branch, this action.js is no longer needed. I removed getDistrictsList function and now this prisma call is executed directly in the components inside DistrictSearch folder.